metriq_gym/run.py

```diff
@@ -20,9 +20,34 @@
 logger.setLevel(logging.INFO)
 
 
+class QBraidSetupError(Exception):
```

Comment:
Do we want to have a separate place to put this exception? We would likely want to use it in other places throughout the code as well.